gapis/api/vulkan/api/renderpass_framebuffer.api

```diff
@@ -180,6 +181,36 @@ cmd VkResult vkCreateRenderPass(
             }
           }
         }
+        case VK_STRUCTURE_TYPE_RENDER_PASS_MULTIVIEW_CREATE_INFO: {
+          ext := as!VkRenderPassMultiviewCreateInfo*(next.Ptr)[0]
+
+          renderPass.MultiviewInfo = new!RenderPassMultiviewCreateInfo(
+            SubpassCount: ext.subpassCount,
+            DependencyCount: ext.dependencyCount,
+            CorrelationMaskCount: ext.correlationMaskCount,
+          )
+
+          if ext.pViewMasks != null {
+            viewMasks := ext.pViewMasks[0:ext.subpassCount]
```

Comment:
can you use the `clone` primitive for all these?

Reply:
Done!